### DISCUSSION TITLE
Missing GitHub Certification Overview

### DISCUSSION THREAD

**Discussion by samtrion** [community/community#101500] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am currently looking for an overview of the GitHub certifications I have already obtained and their validity within my GitHub profile or settings.

Where can this generally be found?

**Comment by BrendaThistle** [comment]
Hi @samtrion The certifications are covered on our [landing page](https://resources.github.com/learn/certifications/). We're working on trying to get achievement badges that folks can add to their profiles.

**Comment by samtrion** [comment]
Hello @BrendaThistle, thank you for your reply. But this is just the general overview. I am asking for an overview of the certifications I have already obtained and the associated information, such as validity etc?
